### PR TITLE
[Merged by Bors] - Implement pause/stop.

### PR DESCRIPTION
**Pull request by razvan**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/hdfs/pages/usage-guide/cluster_operations.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+= Cluster Operation
+
+The managed HDFS instances can be configured with different cluster operations like pausing reconciliation or stopping the cluster. See xref:concepts:cluster_operations.adoc[cluster operations] for more details.
```
Comment (razvan) [inline docs/modules/hdfs/pages/usage-guide/cluster_operations.adoc]:
```suggestion
HDFS installations can be configured with different cluster operations like pausing reconciliation or stopping the cluster. See xref:concepts:cluster_operations.adoc[cluster operations] for more details.
```

File: tests/templates/kuttl/cluster-operation/30-assert.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hdfs-datanode-default
+status:
+  availableReplicas: 0
+  replicas: 0
+---
+apiVersion: v1
+kind: ConfigMap
```
Comment (razvan) [inline tests/templates/kuttl/cluster-operation/30-assert.yaml.j2]: I would remove the ConfigMap asserts

**Review by razvan — COMMENTED** [review]
Some comments.

File: tests/templates/kuttl/cluster-operation/30-assert.yaml.j2 (hunk 2 above)
Comment (maltesander) [reply tests/templates/kuttl/cluster-operation/30-assert.yaml.j2]: https://github.com/stackabletech/hdfs-operator/pull/337/commits/79c69a7b83bf498846d7b50de231e88ef4196091

**Comment by maltesander** [comment]
test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hdfs-operator-it-custom/63/

**Review by maltesander — APPROVED** [review]
bors merge

**Pull request closed without merging**